great_expectations/checkpoint/actions.py

```diff
@@ -168,20 +180,6 @@ def __init__(
         notify_with=None,
         show_failed_expectations=False,
     ) -> None:
```

Comment:
Let's keep some docstring on this and other `__init__` methods, even if just "Create SlackNotificationAction". See https://google.github.io/styleguide/pyguide.html#384-classes for more info about class docstrings.